src/index.ts

```diff
@@ -0,0 +1,29 @@
+import '../assets/style.css';
```

Comment:
file marked as "created" instead of "renamed", will git save history in such case?